entity-framework/core/modeling/relationships.md

```diff
@@ -164,7 +161,7 @@ public class Post
 
 ### Cascade Delete
 
-By convention, cascade delete will be set to *Cascade* for required relationships and *SetNull* for optional relationships. *Cascade* means dependent entities are also deleted. *SetNull* means that foreign key properties in dependent entities are set to null.
+By convention, cascade delete will be set to *Cascade* for required relationships and *Restrict* for optional relationships. *Cascade* means dependent entities are also deleted. *Restrict* means that dependent entities remain unchanged and must be manually deleted, or updated to point to a valid principal entity.
```

Comment:
Maybe also say that the FK can be set to null. Something like, "*Restrict* means that dependent entities remain unchanged and must be manually deleted, or FK properties must be updated to null or to point to a valid principal entity."

Or maybe, since we're talking about delete behavior, just say that they can be set to null. Setting the FK property to point to some other entity is valid for both cascade and restrict. Maybe: "*Restrict* means that untracked dependent entities remain unchanged and must be manually deleted or foreign key properties set to null."

I added "untracked" because it's still a little confusing to me since for tracked entities we will always attempt to set the FK properties to null, so this only applies to entities that are in the database but not tracked.